src/HuffConfig.sol

```diff
@@ -84,6 +87,12 @@ contract HuffConfig {
         return this;
     }
 
+    /// @notice sets whether to broadcast the deployment
+    function set_broadcast(bool broadcast) public returns (HuffConfig) {
```

Comment:
Mind adding a test to `HuffConfig.t.sol` for this function? Bit redundant, but just for coverage.